### PR TITLE
Remove federal accounts added to loadtas workflow

### PR DESCRIPTION
**Pull request by russellbodine**
added cleanup function to remove old federal accounts

### CODE REVIEW

File: usaspending_api/references/reference_helpers.py [hunk 1]
```diff
@@ -5,6 +5,10 @@
 from usaspending_api.accounts.models import FederalAccount, TreasuryAppropriationAccount
 
 
+def remove_federal_accounts(tas_tuple=None):
```
Comment (catherinedevlin) [inline usaspending_api/references/reference_helpers.py]: Could we get a docstring on this?  Just `remove_federal_accounts` doesn't answer many questions (remove which ones?  Remove them why?)

... granted, a single line of code is not hard to read, but still.

Comment (afrasier) [inline usaspending_api/references/reference_helpers.py]: Maybe renaming it to `remove_empty_federal_accounts` would clarify things, as well :)

File: usaspending_api/references/reference_helpers.py [hunk 2]
```diff
@@ -5,6 +5,10 @@
...
+    FederalAccount.objects.filter(treasuryappropriationaccount__isnull=True).distinct().delete()
```
Comment (russellbodine) [inline usaspending_api/references/reference_helpers.py]: this only runs once anyway. (we are removing the bad federal accounts that are currently in db)

File: usaspending_api/references/reference_helpers.py (hunk 1 above)
Comment (russellbodine) [inline usaspending_api/references/reference_helpers.py]: renamed and added docstring

Comment (catherinedevlin) [inline usaspending_api/references/reference_helpers.py]: Renaming was a great suggestion; thanks @afrasier  !

**Pull request merged**